src/styles/styles.js

```diff
@@ -954,11 +955,7 @@ const styles = {
         lineHeight: 16,
     },
 
-    optionAlternateTextCompact: {
-        flexShrink: 1,
-        flexGrow: 1,
-        flexBasis: 'auto',
-    },
+    optionAlternateTextCompact,
```

Comment:
I think much better would be
```suggestion
    optionAlternateTextCompact: {
        flexShrink: 1,
        flexGrow: 1,
        flexBasis: 'auto',
        ...optionAlternateTextPlatformStyles,
    },
```